### PR TITLE
Update cookie banner component to follow convention

### PR DESCRIPTION
**Pull request by igloosi**
- replace content parameter so that either text or html parameters, so
that both can be used
- update template to accept text (as default) or html parameters, where
html is marked as safe
- add functionality to specify html attributes that get appended to the
element
- update YAML file to reflect new parameters
- update table of arguments to reflect changes
- update example in cookie-banner.njk file
- change message container css class:
`.govuk-c-cookie-banner__text` is too specific, so its been updated to
`.govuk-c-cookie-banner__message`

### CODE REVIEW

File: src/components/cookie-banner/cookie-banner.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 variants:
 - name: default
   data:
-    content: GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies</a>
+    html: "GOV.UK uses cookies to make the site simpler. <a href='https://www.gov.uk/help/cookies'>Find out more about cookies</a>"
```
Comment (36degrees) [inline src/components/cookie-banner/cookie-banner.yaml]: Personal preference - I'd invert these quotes so we're using double quotes for attributes consistently.

File: src/components/cookie-banner/cookie-banner.njk [hunk 2]
```diff
@@ -1,8 +1,5 @@
 {% from "cookie-banner/macro.njk" import govukCookieBanner %}
 
-{{- govukCookieBanner(
-  {
-    classes: '',
-    content: 'GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies</a>'
-  }
-) -}}
+{{ govukCookieBanner({
+  "html": "GOV.UK uses cookies to make the site simpler. <a href='https://www.gov.uk/help/cookies'>Find out more about cookies</a>"
```
Comment (36degrees) [inline src/components/cookie-banner/cookie-banner.njk]: Double quotes for attributes here too would be good `</pedantry>`

**Comment by igloosi** [comment]
:)

**Comment by igloosi** [comment]
i get this in the macro then:
"html": "GOV.UK uses cookies to make the site simpler. <a href=\"https://www.gov.uk/help/cookies\">Find out more about cookies</a>"

**Comment by 36degrees** [comment]
Can you not use single quotes for the entire string, then you can use double quotes inside the HTML?

File: src/components/cookie-banner/cookie-banner.yaml (hunk 1 above)
Comment (igloosi) [reply src/components/cookie-banner/cookie-banner.yaml]: i get this in the macro then:
```
"html": "GOV.UK uses cookies to make the site simpler. <a href=\"https://www.gov.uk/help/cookies\">Find out more about cookies</a>"
```

**Comment by 36degrees** [comment]
Sorry, now you've edited your comment, I think that's fine too. Happy with escaping or single quotes.

**Pull request merged**